examples/run_generation.py

```diff
@@ -31,13 +31,18 @@ def get_args():
     parser.add_argument(
         "--max_concurrent_requests", type=int, default=256, help="Max number of concurrent requests"
     )
+    parser.add_argument(
+        "--seed", type=int, default=42, help="Random seed for datasets"
+    )
+
     return parser.parse_args()
 
 
 def read_dataset(
     max_input_length: int,
     total_sample_count: int,
-    model_id: str
+    model_id: str,
+    seed: int,
```

Comment:
I don't see seed is used inside of read_dataset. Did I miss anything?

Reply:
Yes, in order to ensure that the order of the dataset samples obtained each benchmark is the same.